docs/config/lua/config/window_alignment.md

```diff
@@ -0,0 +1,28 @@
+---
+tags:
+  - appearance
+---
+# `window_alignment`
+
+Controls the alignment of the terminal cells inside the window when window size is not a multiple of terminal cell size.
+In other words, this option controls where the additional gap will be.
```

Comment:
This sentence makes sense to you and me, but I think it needs more exposition for other users. Right now it talks about a gap without explaining what gap or where.

It should probably explain about how we have to paint in the excess pixels in that case, and that results in a gap.  Then I think it makes it clearer to say that this option allows the user to control where that gap is.